src/sp_config_scanner.cached.c

```diff
@@ -97,11 +97,11 @@ static void str_dtor(zval *zv) {
 
```

Comment:
While I'm reluctant to change an autogenerated file, since its source (src/sp_config_scanner.re) has been modified as well, it's alright.

Reply:
I can omit this change, and let you regenerate the config, since it's rather impossible to review the change, just it's identical to a local re-generation.

Reply:
Nah, it's ok, the change is reviewable :)